### PR TITLE
Web import

### PR DESCRIPTION
**Pull request by magnus-ISU**
I ripped out the use of `FlutterFileDialog` for imports on mobile. I haven't tested what this looks like. That should probably happen, I will try android now.

### CODE REVIEW

File: lib/app/modules/ImportExport/controllers/import_controller.dart [hunk 1]
```diff
@@ -111,31 +110,20 @@ class ImportController extends GetxController
...
-
-      final js = json.decode(await File(importPath).readAsString())
-          as Map<String, dynamic>;
-      toImport.value = ImportSelections.fromJson(js);
-    } catch (e) {
-      // unawaited(analytics.logEvent(name: Events.ImportFail, parameters: {
-      //   'reason': e.toString(),
-      // }));
+    var result =
+        await FilePicker.platform.pickFiles(type: FileType.custom, allowedExtensions: ['json', 'dungeonpaper']);
```
Comment (chenasraf) [inline lib/app/modules/ImportExport/controllers/import_controller.dart:114]: Is `.dungeonpaper` a relevant extension? I think we only need `json` here
```suggestion
        await FilePicker.platform.pickFiles(type: FileType.custom, allowedExtensions: ['json']);
```

File: lib/app/modules/ImportExport/controllers/import_controller.dart [hunk 2]
```diff
@@ -111,31 +110,20 @@ class ImportController extends GetxController
...
+    if (result == null) {
       Get.rawSnackbar(
-        title: tr.backup.importing.error.title,
-        message: tr.backup.importing.error.message,
+        title: S.current.importFailedTitle,
+        message: S.current.importFailedMessage,
```
Comment (chenasraf) [inline lib/app/modules/ImportExport/controllers/import_controller.dart:118]: I recently updated the translation module, now it should be:

```suggestion
        title: tr.backup.importing.error.title,
        message: tr.backup.importing.error.message,
```

**Review by chenasraf — CHANGES_REQUESTED** [review]
Thanks again @magnus-ISU! Looks good, some small changes necessary and we can merge this.

File: lib/app/modules/ImportExport/controllers/import_controller.dart (hunk 1 above)
Comment (magnus-ISU) [reply lib/app/modules/ImportExport/controllers/import_controller.dart:114]: No, but I feel in a future change it should be. Lots of things make `json` files, and ideally you wouldn't have to confuse those with valid files.

However, this change should happen alongside an export change.

**Comment by magnus-ISU** [comment]
Could you fix the pubspec conflicts? I cannot run the app when I prepare a branch to become a PR, because I don't have the firebase stuff.

**Review by chenasraf — APPROVED** [review]

**Pull request merged**